.github/workflows/publish-pypi.yml

```diff
@@ -0,0 +1,19 @@
+name: pypi
+
+on:
+  push:
+    tags:        
+      - "*.*.*"
+
+jobs:
+  publish_pypi:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Build and publish to pypi
+        uses: JRubics/poetry-publish@v1.16
+        with:
+          python_version: "3.9"
```

Comment:
Why 3.9 instead of 3.10?

Reply:
It is the lowest Python we support so I thought it would be the safest. However it should not make any difference because poetry does everything under the hood.